odbc-api/src/handles/statement.rs

```diff
@@ -368,6 +368,14 @@ pub trait Statement: AsHandle {
             .on_success(|| out)
     }
 
+    /// Returns the number of parameters in an SQL statement.
```

Comment:
"Returns" does not add any meaning here. Just shorten the comment to: "Number of placeholders of a prepared query." Since this is on a statement handle it is worth mentioning that this can only be called after a call to `prepare`.

Reply:
Thanks for your review.   
I feel embarrassed that the code is less than the word count of your reply.:scream: